app/assemblers/assemble_exercise_representations_without_feedback.rb

```diff
@@ -22,7 +22,8 @@ def representations
       mentor:,
       track:,
       status: :without_feedback,
-      criteria: params[:criteria],
+      only_mentored_solutions: !!params[:only_mentored_solutions],
+      criteria: params.fetch(:criteria, ''),
```

Comment:
I don't think this is needed either. I think this can be `params[:criteria]` pending discussion on the other comment around this.